client/src/components/NavBar/NavBar.css

```diff
@@ -0,0 +1,5 @@
+.navBar {
```

Comment:
class names should be all lower case and separated by dashes. I.e. `nav-bar`

Reply:
fixed